views/AppView.js

```diff
@@ -0,0 +1,55 @@
+var AppView = Backbone.View.extend({
+  el: $('body'),
+
+  events: {
+    'click .search-video': 'handleSearchClick',
+    'click .title-link' : 'handleLinkClick'
+  },
+
+  handleSearchClick: function () {
+    var searchTermInput = this.$('.search-form').val();
+    this.model.set('searchTerm', searchTermInput);
+  },
+
+  handleLinkClick: function (e) {
+    var clickedVideoId = $(e.currentTarget).data().id;
+    this.model.updateMainVideo(clickedVideoId)
+  },
+
+  initialize: function () {
+    this.listenTo(this.model.get('videos'), 'reset', this.renderPage);
+    this.listenTo(this.model, 'change:searchTerm', this.fetchOnSearch);
```

Comment:
This line (21) should be in a model. The view shouldn't be concerned about changes on the model if it doesn't have to re-render anything in reaction. The view doesn't need to re-render as the `searchTerm` changes, only when there are new videos.